elasticapm/context/contextvars.py

```diff
@@ -38,9 +38,15 @@
 
 class ContextVarsContext(BaseContext):
     elasticapm_transaction_var = contextvars.ContextVar("elasticapm_transaction_var")
-    elasticapm_span_var = contextvars.ContextVar("elasticapm_span_var")
+    elasticapm_spans_var = contextvars.ContextVar("elasticapm_spans_var", default=[])
```

Comment:
I have a suspicion that this is not OK -- I'm guessing I end up mutating this default. Will need to move to tuples or at least verify that this is OK.